lib/activitylog/hooks/logger.js

```diff
@@ -0,0 +1,14 @@
+const { get } = require('lodash');
+const ActivityLog = require('../../models/activity-log');
+
+module.exports = () => {
+  return model => {
+    return ActivityLog.query().insert({
```

Comment:
We might want to consider transaction-ing the database calls if we're potentially making multiple create/updates per call.